server/models/procedures/trial_balance.sql

```diff
@@ -296,8 +299,24 @@ BEGIN
   GROUP BY fiscal_year_id, period_id, account_id
   ON DUPLICATE KEY UPDATE credit = credit + VALUES(credit), debit = debit + VALUES(debit);
 
+  SET _record_uuid = (SELECT record_uuid FROM stage_trial_balance_transaction LIMIT 1);
```

Comment:
This is the right location to make changes to the code.

The Trial Balance allows a user to post multiple transactions at a singe time, but this will only mark a single record as posted because of the LIMIT 1.  Much better to do:
```sql
UPDATE invoice JOIN stage_trial_balance_transaction ON invoice.uuid = stage_trial_balance_transaction.record_uuid SET invoice.posted = 1;
```
... for both cash, invoices, and vouchers.